### PR TITLE
Changelog 3.3 31/03

### PR DESCRIPTION
**Pull request by laclaro**
Updated changelog.

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -50,6 +60,9 @@ Version 3.3, *2020-XX-XX*
       ContainerAudit module (#2029)
     * Fix a bug which prevented defining an SQL resolver without a password (#2030)
     * Fix missing "position" argument on event import with pi-manage (#2036)
+    * Fix timing issues in tests (#2041)
+    * Fix sorting token table by column (#2111)
+    * Fix errors, which occured due to the updating of the java script components (#2061 #2064 #2114 #2117 #2122 #2125 #2127)
```
Comment (cornelinux) [inline Changelog]: This is no fix. Since it was not broken in version 3.2.2. So we do not need ot mentioned this!

File: Changelog [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+    * New token type: WebAuthn/FIDO2 token is initially supported by privacyIDEA (#1468)
     * New token type: Indexed Secret token allows user
       to authenticate with a pre-known secret that can be
       initialized from the user store. (#1986)
+    * New Event Handler Module: Logging module enables custom event-driven logging (#1580)
 
   Enhancements:
-    * Added the remote serial to the tokeninfo of a remote token to better track
-      authenticated devices (#2031)
+    * API
```
Comment (cornelinux) [inline Changelog]: Everything is an API. We have not used this before. Please revert this.

File: Changelog [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
+      * Added the remote serial to the tokeninfo of a remote token to better track
+        authenticated devices (#2031)
+    * Tokens:
+      * todo: Support unicode in questionnaire token (#2045)
```
Comment (cornelinux) [inline Changelog]: ?? What is this? Why is there a todo?
This was a wrong issue. We did not do anything. Please remove this.

File: Changelog [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
     * Event Handler:
       * The OTP token QR code can now be added not only inline but also as an attachment
         to email notifications (#1226)
     * Policies:
+      * Added a policy to define the allowed characters for PINs (#2049)
```
Comment (cornelinux) [inline Changelog]: This was a fix. The allowed characters did not work according to the documentation.
Please move this to fixes.

File: Changelog [hunk 5]
```diff
@@ -1,19 +1,24 @@
...
+      * Added some policies to limit the number of tokens per user and per type (#1375)
```
Comment (cornelinux) [inline Changelog]: *some* sounds strange. Please do somthing like:

Add policies to limit the number of destinct tokentypes per user.

File: Changelog [hunk 6]
```diff
@@ -24,24 +29,29 @@ Version 3.3, *2020-XX-XX*
...
-      * Added a policy to define the allowed characters for PINs (#2049)
     * WebUI
       * Added the option to filter tokens by tokenrealm (#545)
       * Prior to enrollment of soft tokens, such as HOTP, TOTP and PUSH the user is
         offered with a QR codes to direct him to the Authenticator App stores (#1919).
       * Adding version hashes to WebUI components to avoid working with outdated
         templates (#1871)
-      * Updated bootstrap and JQuery to new versions (#2061)
-      * Rework policy matching (#1691 #2038)
+      * Updated angular.js, bootstrap and JQuery (#830)
```
Comment (cornelinux) [inline Changelog]: Actually it is bootstrap and AngularJS

File: Changelog (hunk 1 above)
[the remark this answers is not in the document]
Comment (cornelinux) [reply Changelog]: If you remove this line (please do), the typo will disappear. ;-)

**Review by cornelinux — CHANGES_REQUESTED** [review]

File: Changelog (hunk 1 above)
Comment (laclaro) [reply Changelog:65]: done

File: Changelog (hunk 2 above)
Comment (laclaro) [reply Changelog:11]: moved to Misc

File: Changelog (hunk 4 above)
Comment (laclaro) [reply Changelog:20]: I was referring to your pull request https://github.com/privacyidea/privacyidea/pull/2051. You added a policy to define the PIN characters. The issue number also appears in "documentation updated".

File: Changelog (hunk 3 above)
Comment (laclaro) [reply Changelog:15]: This is why there was a todo. Done.

File: Changelog (hunk 5 above)
Comment (laclaro) [reply Changelog:21]: okay

File: Changelog (hunk 4 above)
Comment (laclaro) [reply Changelog:20]: Maybe I should remove the issue number here, since the issue itself does not deal with the PIN policy

File: Changelog (hunk 6 above)
Comment (laclaro) [reply Changelog:38]: changed

File: Changelog (hunk 1 above)
[the remark this answers is not in the document]
Comment (laclaro) [reply Changelog]: done

File: Changelog (hunk 4 above)
Comment (laclaro) [reply Changelog]: Added 2051, since github is smart and resolved issue numbers to pull-requests and vice versa

File: Changelog (hunk 1 above)
Comment (cornelinux) [reply Changelog]: @laclaro Thanks! Please not that usually the reviewer should mark the comment as resolved.

**Review by cornelinux — APPROVED** [review]

**Pull request merged**